content/articles/how-to-create-horizontal-scrolling-menu-using-jquery-and-php/index.md

```diff
@@ -0,0 +1,216 @@
+- [Concerning Menu model](#concerning-menu-model)
+- [Advantages of exploitation custom code to execute flat scrolling over](#advantages-of-exploitation-custom-code-to-execute-flat-looking-over)
+- [Hypertext markup language to show menu and result](#hypertext-markup-language-to-show-menu-and-result)
+- [Available modules and libraries to make navigation menus](#available-modules-and-libraries-to-make-navigation-menus)
+- [jQuery script to address horizontal scrolling](#jquery-script-to-address-horizontal-scrolling)
+- [Bring menu objects from db through hypertext preprocessor and mysql](#bring-menu-objects-from-db-through-hypertext-preprocessor-and-mysql)
+- [Creating database](#creating-database)
+- [File-structure](#file-structure)
+- [Final menu result](#final-menu-result)
+- [Conclusion](#conclusion)
+
+### Prerequisites 
+This project requires less or no knowledge in creating menu bars, but a little knowledge will be an added advantage but don't you worry, as I will take you through every step. First, ensure you have a text editor installed on your computer. 
+
+You can choose any text editor you wish with notepad, visual studio code, Atom, Sublime Text, etc.
+Also, you need a web server installed. In our case, we will use XAMP(Apache HTTP Server), but you can choose to download your own choice, such as WAMP server, Nginx Web Server, Lighttpd Web Server, etc.
+
+### Significance of utilizing flat menu
+ * They limit time spent looking over what clients should perform to arrive at their ideal substance/content.
+ * Using horizontal scrolling menus saves vertical page space by setting a unique component.
+ * It keeps the menu thing list from wrapping to the accompanying line, which will be acceptable while thinking about vertical dividing and style.
+ * The best strategy for showing a rundown menu of things in a solitary way without using a scrollbar.
+ * It will be reasonable to make a responsive menu interface viable with web and versatile ports.
+
+### Available modules and libraries to make navigation menus
+There are a lot of available libraries and modules out on the web; nonetheless, in this article, we will utilize Jquery with the blend of PHP and HTML to make our menu bar that intermediate and beginner web developers will understand.
+
+### Advantages of exploitation custom code to execute flat looking over
+1. Own customization will consistently work on the code and limit the quantity of documents size.
+1. Contrasted with the devoted third-party modules, the custom code stream will be straightforward.
+1. The menus are adjusted in focus with the utilization of CSS text-align highlight
+
+### Concerning Menu model
+Our example code will be straightforward and fundamental to make a unique parchment menu to create our unique model. JQuery techniques are utilized to deal with flat scrolling and movement.
+On the snap occasion of these controls, the jQuery vivify() technique is called to move the menu things to and fro.

```

Comment:
I expected you to describe the menu model. What are these Jquery techniques? what's Jquery vivify()?

> Do not assume all your readers understand everything you're discussing!

Reply:
Jquery techniques  /vivify() simply refers to The jQuery methods or functions
 for handling horizontal scrolling and animation.

Reply:
Explain to the reader!